alphadia/constants/default.yaml

```diff
@@ -189,8 +189,23 @@ search_output:
   # can be either "parquet" or "tsv"
   file_format: "tsv"
 
+optimization:
+  order_of_optimization: null
```

Comment:
please comment heavily here on what these parameters mean.

for `order_of_optimization`, state what is expected (list? integers? names?) and give an example (or set the default here)

Reply:
I'll add comments. I am inclined not to set a default for the reasons set out in https://github.com/MannLabs/alphadia/pull/318#discussion_r1726472814.

Reply:
Please let me know if I have some missunderstanding here.

Should the optimiztion order be a debug/experimental config or something the user really wants to change?

If it's encouraged to be configured by users, the order should be independent of the question of target/automatic. for example `order_of_optimization: [('rt','mobility')('ms2')('ms1)]` 

If this is something experimental for us so we can test different combinations, let's mark it like this.

Reply:
My view is that it's something experimental. How should I mark it as such?

Reply:
just add a comment that this is only for expert users?